Wireframe/index.html

```diff
@@ -1,33 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </header>
-    <main>
-      <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
-        <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
-        </p>
-        <a href="">Read more</a>
-      </article>
-    </main>
-    <footer>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </footer>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Wireframe</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <h1>Wireframe</h1>
+  <p class="subtitle">Learn about README files, wireframes, and Git branches</p>
+
+  <div class="container">
+    <div class="card">
```

Comment:
Please use semantic HTML tags in your index.html.
for further information you can refer to this link 
https://www.w3schools.com/html/html5_semantic_elements.asp